includes/AboutUs.php

```diff
@@ -0,0 +1,157 @@
+        </div>
+      </div><br><hr>
+
+    <!-- End Left Column -->
+    </div>
+
+    <!-- Right Column -->
+    <div class="w3-twothird">
+    
+      <div class="w3-container w3-card-2 w3-white w3-margin-bottom">
+        <h2 class="w3-text-grey w3-padding-16"><i class="fa fa-suitcase fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>About eOrg</h2>
+        <div class="w3-container">
+       <h4><i> <center> eOrg is a free open source software intended to reduce loads of paperwork<br> done inter-organisational office-work.</center></i></h4>
+          <hr>
+        </div>
+        <div class="w3-container">
+           <h1 class="w3-text-grey w3-padding-16"><i class="fa fa-certificate fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>Our Intensions :</h1>
+      <h3> <ul><li>  To reduce paperwork in organisations as far as possible.
+
+	<li>To build an enhanced structure for better application management within organisation.
+
+	<li>Proficient and secure way to manage different types of documents/applications within an organisation.
+</ul>
+</h3>
+          <hr>
+        </div>
+        <div class="w3-container">
+       <h2 class="w3-text-grey w3-padding-16"><i class="fa fa-certificate fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>License</h2>
+    eOrg is distributed under the <a href = "https://opensource.org/licenses/MIT">MIT License.</a><br><br>
+      </div>
+      </div>
+
+      <div class="w3-container w3-card-2 w3-white">
+        <div class="w3-container">
+           <img class="w3-round w3-margin-right" src="../image/aboutus.png" style="width:25%;"><span class="w3-opacity w3-large"></span>
+     <div> <ul><li>
+        NABEEL QAISER ,   CSE STUDENT AT IIT PATNA
+      <li>
+        PRADEEP KUMAR ,   CSE STUDENT AT IIT PATNA
+       <li>
+	DEEPAK VERMA  ,   CSE STUDENT AT IIT PATNA
+        </ul>
+        </div> <hr>  </div>
+        <div class="w3-container">
+      
+ <h2 class="w3-text-grey w3-padding-16"><i class="fa fa-certificate fa-fw w3-margin-right w3-xxlarge w3-text-teal"></i>Want To Contribute...</h2>
+      <h4>Fork the repository and commit/create pull requests in develop branch of eOrg.<br>
+         </h4><a href = "https://github.com/pkgodara/eOrg">Click Here..</a> 
+          </h4>
+          
+          <hr>
+        </div>
+        <div class="w3-container">
+      
+        </div>
+      </div>
+
+    <!-- End Right Column -->
+    </div>
+    
+  <!-- End Grid -->
+  </div>
+  
+  <!-- End Page Container -->
+</div>
+</div>
+<footer class="w3-container w3-teal w3-center w3-margin-top">
+<h2><b><i>
+&copy  
+HTML;
+ 
+echo $html;
+ echo date("Y");
+$html = <<<HTML
+  eOrg.
+</h2>
+ Fallow Us on GitHub.<br>
+ <ul><a href = "https://github.com/dkasb">DEEPAK VERMA</a><br> 
+ <a href = "https://github.com/NabeelQaiser">NABEEL QAISER</a><br> 
+ <a href = "https://github.com/pkgodara">PRADEEP KUMAR</a> <br><br></ul>
+ </footer>
+
+</body>
+</html>
+
+HTML;
+echo $html;
+
+
+
+
+
+?>
```

Comment:
__[code_sniffer]__
* A closing tag is not permitted at the end of a PHP file

<!-- Comment by [SideCI](https://sideci.com) -->